lms/djangoapps/discussion_api/views.py

```diff
@@ -274,6 +278,8 @@ def partial_update(self, request, thread_id):
         Implements the PATCH method for the instance endpoint as described in
         the class docstring.
         """
+        if request.content_type != MergePatchParser.media_type:
```

Comment:
Please update the docstring so it's clear what content type is required for the PATCH method.  As an example, see https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/accounts/views.py#L34.